src/Mosaic/Mosaic.interface.ts

```diff
@@ -0,0 +1,8 @@
+import * as React from 'react'
+
+export interface MosaicProps {
+    items: React.ReactNode[]
```

Comment:
Why don't you allow to pass them as children ?

Reply:
Yes you're right